### DISCUSSION TITLE
Bug in voltage / current scale or divider for FURYF4OSD

### DISCUSSION THREAD

**Discussion by julianschweizer** [iNavFlight/inav#6808] [Q&A] [answered]
I am very confused about voltage and current scale in INAV.

Board: Mamba F405 MK2 (HW Rev. 3) 
ESC:  Mamba F40 MK2
Target: FURYF4OSD
INAV Version: 2.6.1

According to diatone website "Voltage Scale" is 110 and "Current Scale" is 115.

**Voltage scale:**

If I set it to 110 the voltage in INAV shows 2,41 V instead of 24,14 V measured with voltmeter. This is why i set voltage scale to 1100 which gives me correct values. I am fine with this.

**Current Scale:**

I tried calibration with "general method" (read charged mAh) described in INAV wiki for >6 times now and i failed to get plausible values. This lead me to start from scratch again: If I measure current draw in idle state I get ~ 170 **m**A out of my current meter. With Current Scale set to 115 (exact 113) I got current draw of 170 **A** in INAV. In theory I need to set the current scale to 115000 (exact 113). But it is possible to set values above 10000 (Need 113000) for current scale in INAV.

Is this a bug? How to set correct scale?

**Comment by julianschweizer** [comment, marked as answer]
I found out that INAV targets are fucked up somehow ... . FURYF4OSD (Betaflight Target for my FC) has RSSI on C2 and CURRENT METER on C3 which should be the other way round. Current sensor is working with "MAMBAF405US", but the rest is not .... . 

It is very unclear which target belongs to which board ... . I open bug report to clarify.

**Reply by SpencerMKSmith** [reply]
Could you by chance link to the bug that you opened?  I'm having this same issue and am very interested in figuring out how to get an accurate current sensor with this board.

**Reply by PhilippMolitor** [reply]
can you name the corresponding issue? I'm having the same problem, didn't find a solution to it so far. (btw: i'm on 4.0.0RC2, still the same problem as you have)

**Comment by avsaase** [comment]
The best way I have found the calibrate the current sensor is to use the process described here: https://www.mrd-rc.com/tutorials-tools-and-testing/flight-controller-therapy/setting-the-current-sensor-in-inav/.
